packages/@blockprotocol/graph/src/types.ts

```diff
@@ -173,6 +173,7 @@ export type EntityType = {
     $schema: string;
     title: string;
     type: string;
+    labelProperty: string;
```

Comment:
```suggestion
    labelProperty?: string;
```
I think we made it required in HASH to remind us to enforce it on predefined schemas, but it should be optional here because it's optional in the spec – apologies, I should've noticed / said this before.

Reply:
Updated